tests/unit/aofrw.tcl

```diff
@@ -179,26 +179,25 @@ start_server {tags {"aofrw external:skip"} overrides {aof-use-rdb-preamble no}}
     }
 
     test {BGREWRITEAOF is delayed if BGSAVE is in progress} {
-        r multi
+        r config set rdb-key-save-delay 1000000
         r bgsave
-        r bgrewriteaof
-        r info persistence
-        set res [r exec]
-        assert_match {*scheduled*} [lindex $res 1]
-        assert_match {*aof_rewrite_scheduled:1*} [lindex $res 2]
+        assert_match {*scheduled*} [r bgrewriteaof]
+        assert_match {*aof_rewrite_scheduled:1*} [r info persistence]
+        r config set rdb-key-save-delay 0
```

Comment:
setting the delay back to 0 is not enough, the fork will keep running (the loop below will still take a long time), we need to kill it `exec kill -9 [get_child_pid 0]`.

p.s. maybe change that loop to also use `[s aof_rewrite_scheduled]`

Reply:
`config set rdb-key-save-delay 1000000`  is only delay one second.

Reply:
yes, but:
1. one second may not be enough, in case the tests are running on a slow or overloaded system, and the test suite isn't scheduled while redis keeps progressing.
2. i rather not delay the test suite for a full 1 second when it runs on a fast system.

Reply:
OK